src/reducers/topicReducer.js

```diff
@@ -0,0 +1,13 @@
+import * as types from '../actions/actionTypes';
+import initialState from './initialState';
+
+export default function topicReducer(state = initialState.topics, action) {
+  switch (action.type) {
+    case types.GET_TOPICS: {
+      const topics = action.topics.topics;
+      return { ...state, values: topics };
```

Comment:
It's not necessary to spread the state(`...state`) because you are already defining all the props of the state(`values`)